tezos/client/tests/ffi_rpc_router_test.rs

```diff
@@ -608,8 +608,8 @@ mod test_data {
     "#;
 
     pub const COMPUTE_PATHS_RESPONSE: &str = r#"
-    {"operations_hashes_path":[{"Left":{"path":{"Left":{"path":"Op","right":[124,9,247,196,215,106,206,134,225,167,225,199,220,10,12,126,220,170,139,40,73,73,50,0,129,19,25,118,168,119,96,195]}},"right":[109,139,183,48,123,47,223,97,128,222,175,250,43,155,217,57,95,136,176,113,167,222,105,2,82,194,14,240,52,251,177,244]}},{"Left":{"path":{"Right":{"left":[124,9,247,196,215,106,206,134,225,167,225,199,220,10,12,126,220,170,139,40,73,73,50,0,129,19,25,118,168,119,96,195],"path":"Op"}},"right":[109,139,183,48,123,47,223,97,128,222,175,250,43,155,217,57,95,136,176,113,167,222,105,2,82,194,14,240,52,251,177,244]}},{"Right":{"left":[10,55,241,142,37,98,174,20,56,135,22,36,123,224,212,228,81,215,44,227,141,29,74,48,249,45,47,110,249,91,73,25],"path":{"Left":{"path":"Op","right":[80,146,47,58,138,154,211,234,76,207,216,93,186,145,204,85,44,17,216,195,123,110,192,172,166,40,175,8,61,159,187,26]}}}},{"Right":{"left":[10,55,241,142,37,98,174,20,56,135,22,36,123,224,212,228,81,215,44,227,141,29,74,48,249,45,47,110,249,91,73,25],"path":{"Right":{"left":[124,9,247,196,215,106,206,134,225,167,225,199,220,10,12,126,220,170,139,40,73,73,50,0,129,19,25,118,168,119,96,195],"path":"Op"}}}}]}
```

Comment:
@adonagy @akoptelov 
Ado, can you please check this json? is this tezos json or it is just our json for comparing?
should we save the old shape?

Reply:
To my understanding we use either FFI or binary encoding to communicate, which both kept intact (they should be the same as before).

Reply:
IIRC, this is a tezos json. Let me double check that

Reply:
@bkontur Just as @akoptelov said. I guess this json was just our own interpretation of the encoding for debugging purposes.

Reply:
> I guess this json was just our own interpretation of the encoding for debugging purposes.
Then I will replace it with just comparing hashes in code.

Reply:
yes, please replace it to hash comparision,

and also, this we would like to reimplement in rust TE-207 a remove this ffi call